### PR TITLE
adding messageHandler config for webCompat

### PR DESCRIPTION
**Pull request by shakyShane**
**Asana Task/Github Issue:**

#### Reference
- [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
- [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
- [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

**Comment by shakyShane** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1155** <a href="https://app.graphite.dev/github/pr/duckduckgo/privacy-configuration/1155" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/privacy-configuration/1155?utm_source=stack-comment).

File: tests/schemas/webcompat-settings.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        "domains": {
+            "description": "List of domains with specific patch settings",
+            "type": "array",
+            "items": {
+                "title": "Domain",
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "domain": {
+                        "type": "string",
```
Comment (jonathanKingston) [inline tests/schemas/webcompat-settings.json:80]: This can also be an array of strings.

File: tests/schemas/webcompat-settings.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                                "op": {
+                                    "type": "string",
+                                    "description": "The operation to be performed"
+                                },
+                                "path": {
+                                    "type": "string",
+                                    "description": "The path of the setting to be patched"
+                                },
+                                "value": {
+                                    "type": "string",
```
Comment (jonathanKingston) [inline tests/schemas/webcompat-settings.json:100]: This is usually an object no?

File: tests/webcompat-tests.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const expect = require('chai').expect
+const fs = require('fs')
+
+describe('WebCompat Settings Tests', () => {
+    const Ajv = require('ajv').default
+    const ajv = new Ajv()
+    const rootSchema = JSON.parse(fs.readFileSync('./tests/schemas/webcompat-settings.json', 'utf8'))
```
Comment (jonathanKingston) [inline tests/webcompat-tests.js:7]: Should this not be checking the file output for each one? We're doing overrides already which isn't captured here.

File: tests/schemas/webcompat-settings.json (hunk 1 above)
Comment (shakyShane) [reply tests/schemas/webcompat-settings.json:80]: done

File: tests/schemas/webcompat-settings.json (hunk 2 above)
Comment (shakyShane) [reply tests/schemas/webcompat-settings.json:100]: for my case, it's just a string. Perhaps we should make it more general when we extract this to be re-used

File: tests/webcompat-tests.js (hunk 3 above)
Comment (shakyShane) [reply tests/webcompat-tests.js:7]: I can change it to just check the output of macos build instead

**Comment by shakyShane** [comment]
@jonathanKingston updated as we spoke about

**Review by jonathanKingston — APPROVED** [review]

**Pull request merged**